_data/collections.yaml

```diff
@@ -4,16 +4,53 @@
 # This is simple, hopefully provides you with a good enough example to make changes as needed
 - name: GSA
   url: https://gsa.gov
-  description: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec dapibus interdum pellentesque. Integer eu vehicula elit. Sed cursus magna in dui suscipit rhoncus. 
+  description: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec dapibus interdum pellentesque. Integer eu vehicula elit. Sed cursus magna in dui suscipit rhoncus.
```

Comment:
what are these descriptions for? do you know? I'm wondering if this is only used in decap.

Reply:
It might even be example data?  I honestly don't know and will defer to @apburnes and you.